package/upgrade/upgrade_node.sh

```diff
@@ -388,6 +420,8 @@ convert_nodenetwork_to_vlanconfig() {
     miimon=0
   fi
 
+  # below code is re-entrancable
```

Comment:
Maybe using the word "idempotent" is more suitable.

Reply:
Reasonable, updated, thanks.